test/test.js

```diff
@@ -1020,7 +1068,8 @@ suite('GaiaHeader', function() {
   });
 
   test('it clears pending async `setTitleStyles`', function(done) {
-    this.dom.innerHTML = '<gaia-header action="menu"><h1>Title</h1></gaia-header>';
+    this.dom.innerHTML = '<gaia-header action="menu"><h1>Title</h1>' +
+      '</gaia-header>';
     var el = this.dom.firstElementChild;
 
     el.runFontFit();
```

Comment:
I think it not worth to break the rules cross gaia, but I could set `jshint maxlen:100` for test files if you prefer that.